### PR TITLE
Vagrantfile: use debian stable instead of testing

### PR DESCRIPTION
**Pull request by tobsan**
This removes the testing upgrade of the VM. It also makes latexpdf
generation of documentation optional.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -1,16 +1,11 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-ram = 4 #GB
```
Comment (frznlogic) [inline Vagrantfile:4]: Actually, maybe we should make this a bit useful instead? set vb.cpus and vb.memory to this X lala etc, and then add environment variables so if you have VAGRANT_RAM=16 you get 16GB ram used for the vagrant machine?

**Review by frznlogic — CHANGES_REQUESTED** [review]
Some changes plix?

File: Vagrantfile (hunk 1 above)
Comment (tobsan) [reply Vagrantfile:4]: I can live with that, I'll make the necessary changes.

Comment (tobsan) [reply Vagrantfile:4]: Updated with changes.

**Pull request merged**